SportsHub/SportsHub.Core/Models/CustomExceptionModels/BusinessLogicException.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace SportsHub.Api.Exceptions.CustomExceptionModels;
+
+public class BusinessLogicException: Exception
+{
```

Comment:
why have you decided to get rid of CoreException?

Reply:
Maybe I misunderstood you , but I thought you didnt wanted Abstract class

Reply:
Okey i will add abstract class

Reply:
So i need to implement a CoreException which inherits the Exception then our CustomExceptions will inherit the Core , but it shoudnt be a abstract class ? Did i get it right

Reply:
yes, so you do not need to duplicate properties

Reply:
okey i will .